spec/acceptance/nodesets/oel.yml

```diff
@@ -9,22 +9,21 @@ HOSTS:
   client:
     roles:
       - default
-      - master
       - client
-    platform: el-8-x86_64
-    box: generic/oracle8
+    platform: el-7-x86_64
```

Comment:
Why did you revert to oel6 and oel7 testing?  oel.yml was testing oel 7 and oel 8.